common/Windows/SecondaryWindow.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Contracts.Services;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Markup;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.WindowsAndMessaging;
+using WinUIEx;
+
+namespace DevHome.Common.Windows;
+
+[ContentProperty(Name = nameof(SecondaryWindowContent))]
+public class SecondaryWindow : WindowEx
```

Comment:
`SecondaryWindow` is now a `Window` instead of an extension file (previously called `SecondaryWindowExtension.cs`). This change allows properties and title bar to be configured from XAML. For more details, check the `SecondaryWindow.md` doc file (in this PR).